subsys/bluetooth/controller/ll_sw/ull_llcp_conn_upd.c

```diff
@@ -209,19 +234,31 @@ static void lp_cu_send_conn_update_ind(struct ll_conn *conn, struct proc_ctx *ct
 		ctx->state = LP_CU_STATE_WAIT_TX_CONN_UPDATE_IND;
 	} else {
 		/* TODO(thoh): Hardcoded instant delta +6 */
-		ctx->data.cu.instant = lp_event_counter(conn) + 6;
+		ctx->data.cu.instant = lp_event_counter(conn) + 6U;
```

Comment:
why not use the ull_conn_event_counter()? 
Also same comment as below re. define for delta